zanata-client-commands/src/main/resources/prompts.properties

```diff
@@ -43,6 +43,8 @@ includes.prompt=Includes (leave blank if not applicable):
 excludes.question=Do you want to define any Excludes?
 excludes.usage= - Wildcard pattern to exclude files and directories. Defined it as "configuration.properties,build.properties".
 excludes.prompt=Excludes (leave blank if not applicable):
+project.file.type.question=What file types do you want to use?
+file.type.prompt=Please enter file types in comma separated format (e.g. XML_DOCUMENT_TYPE_DEFINITION,IDML[txt]):
```

Comment:
Suggest changing `IDML[txt]` to `PLAIN_TEXT[md;markdown]`, because no one would usually name their IDML files with .txt extension, but people do have to use plain text for their markdown files at the moment.